### PR TITLE
Update iphonesimulator version in Travis configuration

### PR DESCRIPTION
**Pull request by adempsey**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @adempsey has signed the CLA. Thanks for the pull request!

File: .travis.yml [hunk 1]
```diff
@@ -5,4 +5,4 @@ before_install:
 podfile: BoxContentSDK/Podfile
 osx_image: xcode8.2.1
 script: 
-  - xcodebuild test -scheme BoxContentSDKTests -sdk iphonesimulator8.4 -project BoxContentSDK/BoxContentSDK.xcodeproj -configuration Debug  | xcpretty -c && exit ${PIPESTATUS[0]}
+  - xcodebuild test -scheme BoxContentSDKTests -sdk iphonesimulator10.2 -project BoxContentSDK/BoxContentSDK.xcodeproj -configuration Debug  | xcpretty -c && exit ${PIPESTATUS[0]}
```
Comment (boxtcarpel) [inline .travis.yml]: can we just set this to 'iphonesimulator' ? That was it will use the latest.

**Review by boxtcarpel — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (adempsey) [reply .travis.yml]: Good call, done

**Review by boxtcarpel — APPROVED** [review]
+2 LGTM

🛬

**Pull request merged**